sunpy/net/tests/test_attr.py

```diff
@@ -344,11 +345,21 @@ def test_attr_numbes():
 
 
 def test_attr_iterable_length():
+
     # not iterable
-    with pytest.raises(ValueError):
+
+    error_message1 = r"Invalid input value: AIA for key: <class 'sunpy\.net\.tests\.test_attr\.Instrument'>\s*sunpy\.net\.tests\.test_attr\.Instrument\s*Dummy Instrument Class\.\s*Attribute Name\s*Client\s*Full Name\s*Description.*?The value is not iterable or just a string\."
```

Comment:
Check for just "The value is not iterable or just a string"

Reply:
Done. Thank You.